components/treeview/stories/treeview.stories.js

```diff
@@ -44,8 +46,149 @@ export default {
 	},
 };
 
+/**
+ * By default, treeviews span the entire width of their parent container and are used within panels.
+ */
 export const Default = TreeViewGroup.bind({});
 Default.args = {
+	items: [
```

Comment:
I changed the default args and some of the other args here, in particular, I wanted to remove selected/disabled states unless they were relevant to the variant/state being displayed.

Reply:
I think this is okay for now, since the Chromatic view of Default still shows at least one disabled item under "Flat markup". Which is the same type of disabled item displayed in the rest of the prod Chromatic view.

I do notice that we need additional disabled item coverage in VRTs. I don't think it's necessary to do as part of this PR, but we can create a followup issue to make sure that we cover:
- Disabled of an expanding item with chevron.
- Disabled of an item with icon (folders and files). Both tiers for good measure.
- Disabled of an item with thumbnail.